src/redux/flights/flights.js

```diff
@@ -18,7 +18,9 @@ export const get = () => async (dispatch) => {
 };
 
 export const post = (data) => async (dispatch) => {
+  console.log(data);
```

Comment:
Can you please remove this line before merging? It causes a warning

Reply:
alright will do thanks